CHANGELOG.md

```diff
@@ -5,6 +5,41 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to the versioning scheme outlined in the [README.md](README.md).
 
+## [2.5.0.0.0]
+
+This release implements the 2.5 Stacks consensus rules: primarily the instantiation
+of the pox-4 contract. For more details see SIP-021.
```

Comment:
It might be worth adding a blurb here about how this is the penultimate release before Nakamoto, which needed to be broken into two consensus-breaking releases in order to stage PoX-4 on mainnet and get signers deployed prior to fast blocks coming online.

Reply:
Related, it would explain why this diff is so massive compared to the things that are actually consensus-visible.  A substantial chunk of the 3.0 code is here; it's just inactive.

Reply:
Also, this section does not mention the activation height.  Perhaps it should?

Reply:
Updated in https://github.com/stacks-network/stacks-core/commit/6d6bd23ec6ef5fcd7f3f74349c96dd00b93ae740